### PR TITLE
Hardcode favicon not found (404) for custom domains

### PR DESCRIPTION
**Pull request by OmarIthawi**
The custom domains nginx configs aren't reading the `nginx_lms_extra_locations` value, so I'm hardcoding the value instead.

**Disclaimer:** I don't fully understand the subject, so that's literally my 8th pull request to attempt to fix it.

Long story on both [Slack](https://appsembler.slack.com/archives/C0DLX9U86/p1565260527047900) and [Trello](https://trello.com/c/oxnXjXv2/4265-2-fix-specific-cases-where-browser-displays-open-edx-logo-as-favicon).

### CODE REVIEW

**Review by thraxil — APPROVED** [review]
This is fine with me. It's up to you whether you want to take this hard-coding approach or pass the variable through.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms-common-settings.j2 [hunk 1]
```diff
@@ -224,9 +224,20 @@ location ~ ^{{ EDXAPP_MEDIA_URL }}/(?P<file>.*) {
 
   {% include "extra_locations_lms.j2" ignore missing %}
 
-  {% if nginx_lms_extra_locations != "" %}
-    {{ nginx_lms_extra_locations }}
-  {% endif %}
+  # Hack: This rule is hardcoded here instead of using `nginx_lms_extra_locations` because of
+  #       `tahoe_certs_agent` wasn't getting the value from `server-vars.yml` and we don't know why.
```
Comment (johnbaldwin) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms-common-settings.j2:228]: > we don't know why.

Yet :)

**Comment by johnbaldwin** [comment]
I'll second @thraxil 's comment. Let's go with this if it addresses the 404 issue for custom domains. Do we want to track this as an outstanding tech debt issue so that we can dig into it more at a future date?

**Review by johnbaldwin — APPROVED** [review]

**Review by iamjazzar — APPROVED** [review]

**Review by melvinsoft — APPROVED** [review]

**Comment by OmarIthawi** [comment]
It's working on production! The new servers were much needed obviously!

cc: @johnbaldwin @ahmedaljazzar @melvinsoft @thraxil!

Closing this PR because it's not needed!

**Pull request closed without merging**